packages/eset_protect/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.3.1"
```

```suggestion
- version: "0.4.0"
```
Enhancements bump minor, bugfixes bump patch.